pkg/logstructured/logstructured.go

```diff
@@ -33,7 +33,9 @@ func (l *LogStructured) Start(ctx context.Context) error {
 	if err := l.log.Start(ctx); err != nil {
 		return err
 	}
-	l.Create(ctx, "/registry/health", []byte(`{"health":"true"}`), 0)
```

Comment:
We specifically don't care about the error here.  You can add a comment or _, _ := to indicate as such